README.md

```diff
@@ -45,6 +45,19 @@ Argo is a [Cloud Native Computing Foundation (CNCF)](https://cncf.io/) hosted pr
 
 ![Screenshot](docs/assets/screenshot.png)
 
+## Try Argo using Nix
+Nix is a package manager / build tool which focusses on reproducible build environments.
+Argo Workflows has some basic support for Nix which is enough to get Argo Workflows up and running with minimal effort. 
+Here are the steps to follow: 
+  1. modify the hosts file according to [this](https://argoproj.github.io/argo-workflows/running-locally/), don't worry about the other instructions. 
+  2. setup a k8s cluster, k3d is the recommended solution here. 
+  3. Install [Nix](https://nixos.org/download.html) 
+  4. Run "nix develop" (you will have to use the experimental feature flags on a fresh install, the features you need are "nix-command" and "flakes").
+  5. Run "devenv up". 
+  6. That is all you need. 
+
+As I said, this is still fairly barebones at the moment, any feature in the Makefile not mentioned here is excluded at the moment. 
+In practice this means only a "make start UI=true" equivalent is supported at the moment, there are no LDFlags set in the build, so the UI will show something similar to "0.0.0-unknown"
```

Comment:
I got to `make start UI=true`, and it fails with `compile: version "go1.20.3" does not match go tool version "go1.19.8"` whilst compiling `dist/workflow-controller`.
I'm on real linux, (arch) and have go installed manually
`PATH="/opt/go/1.20.3/bin:/usr/local/bin:/usr/bin:/home/alan/.krew/bin:/home/alan/.local/bin:/usr/local/sbin:/var/lib/flatpak/exports/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/home/alan/.krew/bin:/home/alan/.local/bin"` before I enter nix, and `GOROOT="/opt/go/1.20.3"` both in and out. I guess GOROOT is causing this.

Reply:
Unsetting GOROOT does the job, so nix should control that environment variable.

Reply:
Thanks @Joibel I shall do that now.